assets/js/modules/analytics/components/common/UACutoffWarning.js

```diff
@@ -85,15 +89,20 @@ export default function UACutoffWarning( { className } ) {
 		! isGA4Connected &&
 		stringToDate( referenceDate ) >= stringToDate( UA_CUTOFF_DATE );
 
+	const eventCategory = `${ viewContext }_widget-ua-stale-warning`;
+
+	useEffect( () => {
+		if ( shouldDisplayWarning ) {
+			trackEvent( eventCategory, 'view_notification' );
+		}
+	}, [ eventCategory, shouldDisplayWarning, viewContext ] );
```

Comment:
`viewContext` seems to be unnecessary here. `eventCategory` includes it and `viewContext` isn't used separately in the effect.

Reply:
Another mistake, thank you for noticing, updated! I assumed ESLint would complain about this, but looks like it doesn't. Thank you!

Reply:
That is odd, is there something wrong with our ESLint config? 🤔

Reply:
I don't think so. I think the config complains about unnecessary dependencies within `useEffect` only if they are "outer scope" variables mutating which doesn't re-render the component.